### DISCUSSION TITLE
控制台设置暂停播放的指令是什么？

### DISCUSSION THREAD

**Discussion by YSYHZ** [hooke007/mpv_PlayKit#371] [Q&A 问答]
翻了翻官方文档，实在没找到。
只找到了`cycle pause`这个循环暂停的指令，如果在暂停的状态下输入这个指令，则会恢复播放。
我想要的指令是将播放状态强制设置为暂停

**Comment by hooke007** [comment]
设置某个属性为xx用的是 `set` 而不是 `cycle`

顺带记得看看手册里对cycle和set这部分的详细说明
